### PR TITLE
notice: add #stash

### PR DESCRIPTION
**Pull request by kyrylo**
See discussion here why it is needed:
https://github.com/airbrake/airbrake/pull/650

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -566,6 +566,20 @@ Accesses a notice's payload, which can be read or filtered. Payload includes:
...
+#### Notice#objects[] & Notice#objects[]=
+
+Each notice can carry arbitrary objects. The `objects` parameter is yielded
+while executing [`add_filter`](#airbrakeadd_filter) callbacks. You can retrieve
+wanted information from attached objects and modify the notice's payload:
+
+```rb
+notice.objects[:my_object] = Object.new
+
+Airbrake.add_filter do |notice, objects|
```
Comment (vmihailenco) [inline README.md]: It is rather strange that you write data to `notice.objects`, but access it by accepting `objects` in filters. Why user can't read from `notice.object` information he wrote there?

File: README.md [hunk 2]
```diff
@@ -566,6 +566,20 @@ Accesses a notice's payload, which can be read or filtered. Payload includes:
...
 ```
 
+#### Notice#objects[] & Notice#objects[]=
+
+Each notice can carry arbitrary objects. The `objects` parameter is yielded
+while executing [`add_filter`](#airbrakeadd_filter) callbacks. You can retrieve
+wanted information from attached objects and modify the notice's payload:
+
+```rb
+notice.objects[:my_object] = Object.new
```
Comment (vmihailenco) [inline README.md]: Probably `notice.stash`?

File: README.md (hunk 1 above)
Comment (kyrylo) [inline README.md]: 👍

File: README.md (hunk 2 above)
Comment (kyrylo) [inline README.md]: Sounds good.

**Comment by kyrylo** [comment]
PTAL

**Pull request merged**